src/systems/ec/implementations/react/components/site-header-core/src/SiteHeaderCore.jsx

```diff
@@ -52,6 +56,53 @@ const SiteHeaderCore = ({
             />
           </a>
           <div className="ecl-site-header-core__action">
+            {!!(
+              loginToggle &&
+              Object.values(loginToggle).length >= 1 &&
+              loginBox
+            ) && (
+              <div className="ecl-site-header-core__login-container">
+                {logged && (
+                  <Fragment>
+                    <a
+                      className="ecl-link ecl-link--standalone ecl-site-header-core__login-toggle"
+                      href={loginToggle.hrefLogged}
+                      data-ecl-login-toggle
+                      aria-controls={loginBox.id}
+                      aria-expanded="false"
+                    >
+                      <Icon shape="general--logged-in" size="s" />
+                      {loginToggle.labelLogged}
+                    </a>
+
+                    <div
+                      id={loginBox.id}
+                      className="ecl-site-header-core__login-box"
+                      data-ecl-login-box
+                    >
+                      <p className="ecl-site-header-core__login-description">
```

Comment:
The div can be conditionally displayed based on presence of `loginBox.description`, if empty string is passed, the div will be rendered empty.